package.json

```diff
@@ -31,17 +31,51 @@
 		"lasttranslator": "Themeisle Translate Team <friends@themeisle.com>"
 	},
 	"dependencies": {
+		"@emotion/react": "^11.4.1",
+		"@wordpress/api-fetch": "^5.2.3",
+		"@wordpress/block-editor": "^7.0.3",
+		"@wordpress/blocks": "^11.1.1",
+		"@wordpress/components": "^18.0.0",
+		"@wordpress/compose": "^5.0.3",
+		"@wordpress/data": "^6.1.1",
+		"@wordpress/date": "^4.2.2",
+		"@wordpress/dom-ready": "^3.2.2",
+		"@wordpress/edit-post": "^5.0.3",
+		"@wordpress/element": "^4.0.2",
+		"@wordpress/eslint-plugin": "^9.2.0",
+		"@wordpress/hooks": "^3.2.1",
+		"@wordpress/html-entities": "^3.2.2",
+		"@wordpress/i18n": "^4.2.3",
+		"@wordpress/icons": "^4.1.0",
+		"@wordpress/plugins": "^4.0.3",
+		"@wordpress/scripts": "^18.0.1",
+		"@wordpress/server-side-render": "^3.0.3",
```

Comment:
Why are we installing so many dependencies when they already exist in the core? This will make the build files really big as those scripts aren't small. We should keep using externals.

Reply:
They are not included in the build; it is just for dev. WordPress has a plugin that automated the externals.  Also, they are good for autocomplete and auto import.

Reply:
It was initially on the devDeps, but the EsLint complied about it. Might change after some EsLint configuration.